### ISSUE TITLE
Duplicate eslint errors

### ISSUE THREAD

**Issue by ashrafhadden** [vuejs/eslint-plugin-vue#593]
**Tell us about your environment**
VSCode 1.27.2
MacBook Pro (Retina, Mid 2012)

_(tested on both versions)_
* **ESLint Version:** 4.19.1, 5.6.1
* **eslint-plugin-vue Version:** 4.7.1, master
* **Node Version:** 8.9.3, 10.11.0

**Please show your full configuration:**

```js
// .eslintrc.js

module.exports = {
  root: true,
  env: {
    node: true,
  },
  extends: ['plugin:vue/recommended'],
  parserOptions: {
    parser: 'babel-eslint',
  },
}
```

**What did you do? Please include the actual source code causing the issue.**
![image](https://user-images.githubusercontent.com/32565770/46319883-c7fa1380-c5a9-11e8-89a3-d94621da1060.png)

```vue
    <GlobalEvents
      :filter="event => event.target.tagName !== 'INPUT' && 'TEXTAREA'"
      @keydown.digit1="changeActiveTab(1)"
      @keydown.50="test"
    />
```

**What did you expect to happen?**
Only report a single eslint error for `vue/valid-v-on`

**What actually happened? Please include the actual, raw output from ESLint.**
Reports a 2nd duplicate `vue/valid-v-on` error, one from the usual `eslint` source and another from `eslint-plugin-vue` source which does not react to overrides from my `.eslintrc.js`. The normal behaviour is for only one error to appear from the `eslint` source as is obvious from the `vue/attributes-order` error that only shows up once.

**Comment by ota-meshi** [comment]
@ashrafhadden Hello!
I think that errors starting with `eslint-plugin-vue` are reported from the` Vetur` plugin.

I think that you can avoid this issue by making the following settings.

>  - If you use `Vetur` plugin in VSCode - set `"vetur.validation.template": false` to avoid default Vetur template validation. Check out [vetur documentation](https://github.com/vuejs/vetur/blob/master/docs/linting-error.md) for more info.

https://github.com/vuejs/eslint-plugin-vue#why-doesnt-it-work-on-vue-file

**Comment by ashrafhadden** [comment]
AHHHH!!! OMG, TY @ota-meshi!  🙏  xD That totally fixed it :grin:

**Comment by armano2** [comment]
@michalsnik can you close/resolve this one

**Comment by mysticatea** [comment]
Closing as resolved. Thank you!
